api/sentry/utils.py

```diff
@@ -18,6 +18,8 @@ def get_all_future_versions():
 def insert_rates(json_data, csv_file):
     all_future_versions = get_all_future_versions()
     print(all_future_versions)
+    LOW_CRASH_FREE_RATE_THRESHOLD = 99.5
```

Comment:
Before we add this, since it's likely there's other rules like these it would make sense to start making a rules manifest (e.g, YAML) for Android and iOS, and load these static rules in. For example, we don't know for certain if this is applicable (maybe?) for Android and iOS. It doesn't make sense to define them in this function.